configs/SKYSAKURAH7/config.h

```diff
@@ -0,0 +1,220 @@
+#define SPI1_SDI_PIN         PA6
+#define SPI1_SDO_PIN         PA7
+
+//external spi
+#define SPI2_SCK_PIN         PB13
+#define SPI2_SDI_PIN         PB14
+#define SPI2_SDO_PIN         PB15
+
+//spi4 for icm42688
+#define SPI4_SCK_PIN         PE12
+#define SPI4_SDI_PIN         PE13
+#define SPI4_SDO_PIN         PE14
+
+//adc
+#define ADC_VBAT_PIN         PC3
+#define ADC_CURR_PIN         PC2
+#define ADC_EXTERNAL1_PIN    PC0
+#define ADC_RSSI_PIN    PC1
+
+//sd card
+#define SDIO_CK_PIN          PC12
+#define SDIO_CMD_PIN         PD2
+#define SDIO_D0_PIN          PC8
+#define SDIO_D1_PIN          PC9
+#define SDIO_D2_PIN          PC10
+#define SDIO_D3_PIN          PC11
+
+//pinio
+#define PINIO1_PIN           PE15
+#define PINIO2_PIN           PD10
+#define PINIO3_PIN           PD11
+
+//sensor CS and EXTI
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_2_EXTI_PIN      PB2
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_CS_PIN        PE11
+#define BARO_CS_PIN        PC5
+
+/* CS1 pads for SPI2 connection:
+ *
+ * CS1 PB12
+ */
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 1, PB1 , 2,  1) \
+    TIMER_PIN_MAP( 2, PA0 , 2,  2) \
+    TIMER_PIN_MAP( 3, PA1 , 2,  3) \
+    TIMER_PIN_MAP( 4, PA2 , 2,  4) \
+    TIMER_PIN_MAP( 5, PA3 , 2,  5) \
+    TIMER_PIN_MAP( 6, PD12, 1,  6) \
+    TIMER_PIN_MAP( 7, PD13, 1,  7) \
+    TIMER_PIN_MAP( 8, PD14, 1, 12) \
+    TIMER_PIN_MAP( 9, PD15, 1, -1) \
+    TIMER_PIN_MAP(10, PE5 , 1,  0) \
+    TIMER_PIN_MAP(11, PE6 , 1, -1) \
+    TIMER_PIN_MAP(12, PA8 , 1, 14) \
+    TIMER_PIN_MAP(13, PA15, 1,  0) \
+    TIMER_PIN_MAP(14, PC7 , 2,  0) \
+    TIMER_PIN_MAP(15, PC6 , 2,  0) \
+    TIMER_PIN_MAP(16, PB8 , 1,  0) \
+    TIMER_PIN_MAP(17, PB9 , 1,  0)
+
+
+
+#define ADC1_DMA_OPT        8
+#define ADC3_DMA_OPT        9
+#define TIMUP1_DMA_OPT      0
+#define TIMUP2_DMA_OPT      0
+#define TIMUP3_DMA_OPT      2
+#define TIMUP4_DMA_OPT      1
+#define TIMUP5_DMA_OPT      0
+#define TIMUP8_DMA_OPT      0
+
+
+#define USB_DETECT PD4
+
+
+//mag config
+#define MAG_I2C_INSTANCE I2CDEV_1
+#define MAG_I2C_ADDRESS 14
+#define MAG_ALIGN CW0_DEG_FLIP
+
+//default behavior
+#define DEFAULT_RX_FEATURE FEATURE_RX_SERIAL
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_SDCARD
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SCALE 170
+#define DEFAULT_VOLTAGE_METER_DIVIDER 5
+#define DEFAULT_CURRENT_METER_SCALE 250
+#define BEEPER_INVERTED
+#define BEEPER_PWM_HZ 2500
+
+
+//sdcard config
+#define SDCARD_DETECT_PIN NONE
+#define SDIO_DEVICE SDIODEV_1
+#define SDIO_USE_4BIT 1
+
+//pinio config
+#define PINIO1_BOX 40
+#define PINIO2_BOX 41
+#define PINIO3_BOX 43
+#define PINIO1_CONFIG 129
+
+//gyro config
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW270_DEG_FLIP
+#define GYRO_2_SPI_INSTANCE SPI4
+#define GYRO_2_ALIGN CW0_DEG_FLIP
+
+//baro config
+#define BARO_SPI_INSTANCE SPI1
```

Comment:
:thinking:

Reply:
Gyro 2 is IIM42652 which is not supported by bf. This was present because it was  previously a MPU6000, will also delete later.

Reply:
Why not remove gyro 2 defines altogether then?